datax-iotdb-connector/iotdbwriter/doc/iotdbwriter.md

```diff
@@ -0,0 +1,94 @@
+# IOTDBWriter 插件文档
```

Comment:
Hi, please move this UserGuide to docs/UserGuide/zh/Echosystem Integration and add an English version in docs/UserGuide/Echosystem Integration

Reply:
Besides, when we add a new doc, an entry need to added in config.js in site module, please refer to 
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=199539076